truss/remote/baseten/core.py

```diff
@@ -48,6 +48,21 @@ class ChainDeploymentHandle(typing.NamedTuple):
     is_draft: bool
 
 
+class PatchState(typing.NamedTuple):
+    current_hash: str
+    current_signature: str
+
+
+class TrussPatches(typing.NamedTuple):
+    django_patch_state: PatchState
+    container_patch_state: PatchState
+
+
+class TrussWatchState(typing.NamedTuple):
```

Comment:
nit: seems to me that we could model these types in a similar way to the backend

```
class TrussWatchState
     is_container_built_from_push: bool
    container_patch_state: PatchState
    ...
```

why do things differently?

Reply:
this way plays nicer with mypy - you ensure that both of pieces exist which more accurate reflects the different states

{bool = False, empty state} vs {bool = True, present state} I can update the server side response to just be this - lmk what you think